pkg/volume/plugins.go

```diff
@@ -66,6 +66,13 @@ const (
 	CSIVolumePublished CSIVolumePhaseType = "published"
 )
 
+var (
+	deprecatedVolumeProviders = map[string]string{
+		"kubernetes.io/cinder":  "The Cinder volume provider is deprecated and will be removed in a future release",
```

Comment:
Are there plans for cinder or scaleio to implement csi migration?

Reply:
I believe that @adisky and @ramineni would know the most about csi migration plans.

Reply:
@msau42 yes! https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L409